CMakeLists.txt

```diff
@@ -3,7 +3,17 @@ project(Store)
 
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-find_package(GTest REQUIRED)
+# find_package(GTest REQUIRED)
+
+include(FetchContent)
```

Comment:
Yeah this looks better. Does the zip file contain source files or compiled libraries/executables? I am trying to determine if we want this to be completely platform agnostic.

Reply:
It contains precompiled binaries and executables. Can you try running this patch once I merge it in on your machine? That way we can identify if there are any platform specific things we need to worry about

Reply:
Looks like all my tests were able to pass, and could link against Gtest correctly.